src/hooks/useAddRpcEndpoint.ts

```diff
@@ -46,22 +30,25 @@ export function useAddRpcEndpoint() {
       blockExplorerUrl: blockExplorerUrls,
     } = CONFIG.rpc;
     
-    return connectedProvider
-      .send("wallet_addEthereumChain", [
+    console.log('[useAddRpcEndpoint] Add', rpcUrl)
```

Comment:
Should we remove the console.log?

Reply:
Will do. For now this app is not too verbose, so i thought it was okeish. But will remove